canvas_modules/common-canvas/src/notification-panel/notification-panel.jsx

```diff
@@ -106,31 +120,87 @@ class NotificationPanel extends React.Component {
 		}
 	}
 
+	deleteNotification(messageKey) {
+		this.props.canvasController.deleteNotificationMessages(messageKey);
+	}
```

Comment:
This seems a little strange. The method name is singular but the method it's calling is plural with the same parmeter passed through and the parameter name is singular as well. Are these keys or IDs that are being passed here?  Is there one key (ID) or many?

Reply:
The "key" I was generating is no longer used, it's just the message ID now, I've updated the parameter to be clearer.

In the notification panel itself, this function is used only for dismissing individual messages, hence the singular name. I wanted to make the higher level CanvasController function a bit more flexible in case we have other cases in the future, but in the panel itself, deleting individual messages and the clear all button covers all the use cases